frontend/amundsen_application/static/js/components/Table/index.tsx

```diff
@@ -414,6 +447,43 @@ const ShimmeringBody: React.FC<ShimmeringBodyProps> = ({
   </tr>
 );
 
+type ExpandCollapseAllButtonProps = {
+  shouldExpandAllRows?: boolean;
+  hasRowsToExpand?: () => boolean;
+  toggleExpandingRows?: () => void;
+};
+const ExpandCollapseAllButton: React.FC<ExpandCollapseAllButtonProps> = ({
+  shouldExpandAllRows,
+  hasRowsToExpand,
+  toggleExpandingRows,
+}: ExpandCollapseAllButtonProps) => {
+  const buttonContainerStyle = {
+    width: `${getExpandingButtonWidth(0)}px`,
+  };
+
+  return (
+    <span
+      className="ams-table-expanding-button-container"
+      style={buttonContainerStyle}
+    >
+      {hasRowsToExpand && hasRowsToExpand() && (
+        <button
+          type="button"
+          className="ams-table-expanding-button"
+          onClick={toggleExpandingRows}
+        >
+          <span className="sr-only">{EXPAND_ROW_TEXT}</span>
+          {shouldExpandAllRows || shouldExpandAllRows === undefined ? (
+            <UpIcon key="collapseAllIcon" size={IconSizes.SMALL} />
+          ) : (
+            <DownIcon key="expandAllIcon" size={IconSizes.SMALL} />
```

Comment:
I feel these should be different icons.
Maybe double chevrons up and down? Or + / -?